src/map/ai/helpers/gambits_container.cpp

```diff
@@ -912,4 +922,18 @@ namespace gambits
         });
         return hasHealer;
     }
+    // used to check for tanks in party (Volker, AA Hume)
+    bool CGambitsContainer::PartyHasTank()
+    {
+        bool hasTank = false;
+        static_cast<CCharEntity*>(POwner->PMaster)->ForPartyWithTrusts([&](CBattleEntity* PMember) {
```

Comment:
Formatting lamdas is hard and broken:

Drop the curly brace onto the next line, surround the whole statement with // clang-format off/on

Reply:
Here's an example:
```cpp

                // clang-format off
                PMaster->ForParty([](CBattleEntity* PMember)
                {
                    ((CCharEntity*)PMember)->PLatentEffectContainer->CheckLatentsPartyAvatar();
                });
                // clang-format on
```